resources/syntax/CSS (Twig).sublime-syntax

```diff
@@ -16,6 +16,17 @@ contexts:
     - include: Twig.sublime-syntax#twig_embedded
     - include: Twig.sublime-syntax#comment
 
+  main:
+    - meta_prepend: true
+    - meta_include_prototype: false
+    - include: Twig.sublime-syntax#twig_embedded
+    - include: Twig.sublime-syntax#comment
+
+  block-end:
+    - meta_append: true
+    - include: Twig.sublime-syntax#twig_embedded
+    - include: Twig.sublime-syntax#comment
+
```

Comment:
For compatibility reasons with "older" ST builds, the following might needed as well.

```suggestion

  at-rule-block-body:
    # required until ST4173
    - meta_prepend: true
    - meta_include_prototype: false
    - include: Twig.sublime-syntax#twig_embedded
    - include: Twig.sublime-syntax#comment
```